src/services/twilio-api/api-browser.js

```diff
@@ -9,10 +9,31 @@ const OPERATIONS = ['post', 'get', 'delete'];
 class TwilioApiBrowser {
   constructor(spec) {
     spec = spec || this.loadApiSpecFromDisk();
-
+    spec = this.mergeVersions(spec);
     this.domains = this.loadDomains(spec);
   }
 
+  mergeVersions(spec) {
```

Comment:
A test with a truncated `spec` input may be helpful here for the `mergeVersions` function.